### PR TITLE
DVX-8406 Add CD to Jarvis

### PR DESCRIPTION
**Pull request by ifosch**
This implements Jarvis CI/CD in a single Pipeline.

Refs [DVX-8406](https://mydevex.atlassian.net/browse/DVX-8406)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -31,7 +31,36 @@ pipeline {
         success {
             script {
                 if (env.BRANCH_NAME == 'master' ) {
-                    build job: 'jarvis-deploy', wait: false
+                    sh 'sudo pip install awscli'
```
Comment (miguelbernadi) [inline Jenkinsfile]: This is no longer needed.

Comment (ifosch) [reply Jenkinsfile]: Fixed

**Comment by miguelbernadi** [comment]
+1

**Comment by bbrassart** [comment]
+1

**Pull request merged**